tests/integration/024_cloud_credentials/main.tf

```diff
@@ -19,9 +24,12 @@ data "env0_cloud_credentials" "aws_credentials" {
   credential_type = "AWS_ASSUMED_ROLE_FOR_DEPLOYMENT"
 }
 
-data "env0_aws_credentials" "aws_credentials" {
-  for_each = toset(data.env0_cloud_credentials.aws_credentials.names)
-  name     = each.value
+data "env0_aws_credentials" "aws_credentials1" {
+  name     = data.env0_cloud_credentials.aws_credentials.names[index(data.env0_cloud_credentials.aws_credentials.names, env0_aws_credentials.aws_cred1.name)]
+}
+
+data "env0_aws_credentials" "aws_credentials2" {
+  name     = data.env0_cloud_credentials.aws_credentials.names[index(data.env0_cloud_credentials.aws_credentials.names, env0_aws_credentials.aws_cred2.name)]
```

Comment:
First time seeing this format. does it work like `for_each`?

Reply:
nope its the opposite ,
you take a list and search in the list for a specific value

Reply:
@liranfarage89 same as https://github.com/env0/terraform-provider-env0/pull/697